### ISSUE TITLE
Load Dataset #2

### ISSUE THREAD

**Issue by brian-stinar** [codeforabq/traffic-analysis#6]
[1994-2014 Primary & Secondary Coal.xlsx](https://github.com/codeforabq/traffic-analysis/files/801385/1994-2014.Primary.Secondary.Coal.xlsx)

**Comment by deepdivedylan** [comment]
I'm on ya lasagna!

**Comment by tgebhart** [comment]
Use the below for lat/lon (via columns `WGS_X`, `WGS_Y`) instead of UTM

[1994-2014.Primary.Secondary.Coal.Updated.xlsx](https://github.com/codeforabq/traffic-analysis/files/801692/1994-2014.Primary.Secondary.Coal.Updated.xlsx)

**Comment by deepdivedylan** [comment]
Cleansed CSV file committed
https://github.com/codeforabq/traffic-analysis/commit/3c60eb2e9e548ba2ff8c4830a968b98b0432b2f2

**Comment by brian-stinar** [comment]
This is dataset2 inside datasets.

I'm assigning this to @ValeMeza to take care of with his new dataset organization schema.

**Comment by brian-stinar** [comment]
This appears to be loading in dataSources.sql, with some errors. I will adjust dataSources.sql and once running, close this out.

**Comment by brian-stinar** [comment]
No, dataSources.sql is only the sources. I don't see anything loading "1994-2014.Primary.Secondary.Coal.Updated.xlsx"
